mmv1/templates/terraform/examples/bigtable_materialized_view.tf.tmpl

```diff
@@ -0,0 +1,34 @@
+resource "google_bigtable_instance" "instance" {
+  name = "{{index $.Vars "instance_name"}}"
+  cluster {
+    cluster_id   = "cluster-1"
+    zone         = "us-central1-a"
+    num_nodes    = 3
+    storage_type = "HDD"
+  }
+
+  deletion_protection  = {{index $.Vars "deletion_protection"}}
+}
+
+resource "google_bigtable_table" "table" {
+  name          = {{index $.Vars "table_name"}}
+  instance_name = google_bigtable_instance.instance.name
+
+  column_family {
+	family = "CF"
+  }
+}
+
+resource "google_bigtable_materialized_view" "materialized_view" {
```

Comment:
This should use the primary resource id so that tests can identify which resource is under test.

```suggestion
resource "google_bigtable_materialized_view" "{{$.PrimaryResourceId}}" {
```